logic/src/commonMain/kotlin/com/wire/kalium/logic/data/team/TeamRepository.kt

```diff
@@ -80,9 +81,35 @@ internal class TeamDataSource(
         teamsApi.getTeamInfo(teamId = teamId.value)
     }.map { teamDTO ->
         teamMapper.fromDtoToEntity(teamDTO)
-    }.flatMap { teamEntity ->
-        wrapStorageRequest { teamDAO.insertTeam(team = teamEntity) }.map {
-            teamMapper.fromDaoModelToTeam(teamEntity)
+    }.map { teamEntity ->
+        teamDAO.insertTeam(team = teamEntity)
```

Comment:
What if `insertTeam` fails? Errors won't be propagated.

I think the previous version addressed this properly.